python/requirements/ml/dl-gpu-requirements.txt

```diff
@@ -1,5 +1,10 @@
 # If you make changes below this line, please also make the corresponding changes to `dl-cpu-requirements.txt`!
 
+tensorflow==2.11.0; sys_platform != 'darwin' or platform_machine != 'arm64'
+tensorflow-macos==2.11.0; sys_platform == 'darwin' and platform_machine == 'arm64'
+tensorflow-probability==0.19.0
+tensorflow-datasets
```

Comment:
Curious - why are these needed?

Reply:
We've changed the way we install the requirements. Previously we installed the CPU versions first and then the GPU versions on top, but it's more efficient (e.g. for caching) to just install GPU.